<launch>
<param name="use_sim_time" type="bool" value="true" />
<include file="$(find object_detect)/launch/depth_to_pointcloud.launch"/>
<node pkg="object_detect" type="slam_node" name="slam_node" output="screen" cwd="node">
	<param name="icp_max_iters" type="int" value="30"/>
	<param name="max_corr_dist" type="double" value="0.025"/>
	<param name="transform_eps" type="double" value="1e-8"/>
	<param name="voxex_grid_leaf_size" type="double" value="0.08"/>
	<param name="filt_feature_num" type="int" value="30" />
	<param name="min_feature_num" type="int" value="50" />
	<param name="full_match" type="bool" value="true" />
</node>
</launch>
